src/theory/uf/eq_proof.h

```diff
@@ -182,14 +182,44 @@ class EqProof
    * themselves).
    * @return True if the EqProof transitivity step is in either of the above
    * cases, symbolizing that the ProofNode justifying the conclusion has already
-   * beenproduced.
+   * been produced.
    */
   bool expandTransitivityForDisequalities(
       Node conclusion,
       std::vector<Node>& premises,
       CDProof* p,
       std::unordered_set<Node, NodeHashFunction>& assumptions) const;
 
+  /** Expand coarse-grained transitivity steps for theory disequalities
+   *
+   * Currently the equality engine can represent disequality reasoning of theory
+   * symbols in a rather coarse-grained manner with EqProof. This is the case
+   * when EqProof is
+   *            (= t1 c1) (= t2 c2)
```

Comment:
Missing paren.

Reply:
I'm not sure I understand.

Reply:
Oh, I was confused I thought it was equality between equalities, guess not.